lexicons/app/bsky/graph/getLists.json

```diff
@@ -0,0 +1,33 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.getLists",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Fetch a list of lists that belong to an actor",
+      "parameters": {
+        "type": "params",
+        "required": ["actor"],
+        "properties": {
+          "actor": {"type": "string", "format": "did"},
```

Comment:
We've often been using the `at-identifier` format for this kind of parameter (e.g. see `getFollowers`, `getAuthorFeed`).